From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 27 Jun 2015 11:08:08 +0800
Subject: [PATCH] {MAYBE UPS} extcon: remove reference to unknown enum extcon

According to commit 2a9de9c0f08d ("extcon: Use the unique id for
external connector instead of string") it is intended.

https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=2a9de9c0f08d61fbe3764a21d22d0b72df97d6ae
---
 include/linux/extcon/extcon-adc-jack.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/extcon/extcon-adc-jack.h b/include/linux/extcon/extcon-adc-jack.h
index a0e03b13b449..2aa32075bca1 100644
--- a/include/linux/extcon/extcon-adc-jack.h
+++ b/include/linux/extcon/extcon-adc-jack.h
@@ -59,7 +59,7 @@ struct adc_jack_pdata {
 	const char *name;
 	const char *consumer_channel;
 
-	const enum extcon *cable_names;
+	const unsigned int *cable_names;
 
 	/* The last entry's state should be 0 */
 	struct adc_jack_cond *adc_conditions;
-- 
